docs/advanced-features/linked-containers/index.md

```
@@ -55,7 +55,21 @@ services:
     image: postgres
 ```
 
-The `com.docker.compose.depends_on` label is automatically set by Docker Compose and parsed by Watchtower to extract service names.
+The `com.docker.compose.depends_on` label is automatically set by Docker Compose and parsed by Watchtower to extract service names and support implicitly restarting linked services.
+
+!!! Important "Docker Compose depends_on Restart Behavior"
+    Docker Compose's [`depends_on`](https://docs.docker.com/reference/compose-file/services/#depends_on) has a `restart` attribute in the long-form syntax:
+
+    ```yaml
```

markdownlint-cli2 (0.21.0)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)